### PR TITLE
[DOCS] Repository split explained

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Developer/MonoRepository.rst [hunk 1]
```diff
@@ -32,3 +32,25 @@ It is recommended to run the validation check before you commit:
     make test-monorepo
 
 .. _`monorepo-builder`: https://github.com/symplify/monorepo-builder
+
+..  _repository_split:
+
+Repository split
+================
+
+To be made available on packagist each package in the folder packages has
```
Comment (brotkrueml) [inline Documentation/Developer/MonoRepository.rst:41]:
```suggestion
To be made available on Packagist each package in the folder packages has
```

File: Documentation/Developer/MonoRepository.rst [hunk 2]
```diff
@@ -32,3 +32,25 @@ It is recommended to run the validation check before you commit:
...
 
 .. _`monorepo-builder`: https://github.com/symplify/monorepo-builder
+
+..  _repository_split:
+
+Repository split
+================
+
+To be made available on Packagist each package in the folder packages has
+to have its own repository. On merging and creating tag these repositories
```
Comment (brotkrueml) [inline Documentation/Developer/MonoRepository.rst:42]: I don't understand the sentence: "On merging ..."

File: Documentation/Developer/MonoRepository.rst [hunk 3]
```diff
@@ -32,3 +32,25 @@ It is recommended to run the validation check before you commit:
...
+updated by the github action :file:`.github/workflows/split-repositories.yaml`.
+
+To add a new package to the repository split the following is needed:
+
+*   Create a repository with the composer name, special signs replaced by minus.
```
Comment (brotkrueml) [inline Documentation/Developer/MonoRepository.rst:47]:
```suggestion
*   Create a repository with the Composer name, special signs replaced by minus.
```

File: Documentation/Developer/MonoRepository.rst [hunk 4]
```diff
@@ -32,3 +32,25 @@ It is recommended to run the validation check before you commit:
...
+*   Add the repository with :guilabel:`maintain` rights to the group
+    https://github.com/orgs/TYPO3-Documentation/teams/php-based-rendering-bot
+*   Add an entry to file :file:`config.subsplit-publish.json`.
+*   Push some change within the package to main.
+*   Add the new repository to packagist
```
Comment (brotkrueml) [inline Documentation/Developer/MonoRepository.rst:52]:
```suggestion
*   Add the new repository to Packagist.
```

File: Documentation/Developer/MonoRepository.rst [hunk 5]
```diff
@@ -32,3 +32,25 @@ It is recommended to run the validation check before you commit:
...
+
+..  note::
+    Trigger a **push** first before a release is made, to init the split-repo.
+    Or you will get a "object not found".
```
Comment (brotkrueml) [inline Documentation/Developer/MonoRepository.rst:56]:
```suggestion
    Or you will get an "object not found".
```

**Review by garvinhicking — APPROVED** [review]
Made some changes and tried to incorporate Chris' suggestions, hope it is clearer now?

Thanks for getting this off the ground, we need this in our docs to not face the problem of adding new sub-packages again.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**